packages/protocol/script/DeployAndChangeTaikoTokenOnL2.s.sol

```diff
@@ -0,0 +1,62 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.24;
+
+import "../test/DeployCapability.sol";
+import "../contracts/tko/BridgedTaikoToken.sol";
+import "../contracts/tokenvault/ERC20Vault.sol";
+
+contract DeployTaikoToken is DeployCapability {
+    // If the private key to deploy proxy shall be different to the private key to changing the
+    // vault's bridge address(changeBridgedToken) then we need 2 private keys.
+    uint256 public privateKey = vm.envUint("PRIVATE_KEY");
```

Comment:
the privteKey and the current vault owner must match, otherwise,  `changeBridgedToken` will fail. So I suggest to rename these two vars: `vaultOwnerPrivateKe` and `vaultOwner`